samples/subsys/power/stm32_pm/lowpower/src/main.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sample.h"
+
+#define WITHIN_ERROR(var, target, epsilon)       \
+		(((var) >= (target)) && ((var) <= (target) + (epsilon)))
+
+struct device *gpio_port, *gpio_port0, *gpio_port1, *gpio_port2;
+
+struct k_timer lptimer;
+
+/* 1000 msec = 1 sec */
+#define WAIT_TIME 4000
+#define SLEEP_TIME 9000
+
+void button_pressed(struct device *gpiob, struct gpio_callback *cb,
+		    u32_t pins)
+{
+		LL_PWR_ClearFlag_WU();
+		LL_PWR_DisableWakeUpPin(LL_PWR_WAKEUP_PIN2);
+		gpio_pin_write(gpio_port1, LED1, LED_ON);
+		k_busy_wait(100);
+		gpio_pin_write(gpio_port1, LED1, LED_OFF);
+}
+
+static struct gpio_callback gpio_cb;
+
+
+/* Application main Thread */
+void main(void)
```

Comment:
A README would be nice to allow to undertsand quickly what the sample is supposed to do

Reply:
comment added to the main.c